test/NuGet.Clients.FuncTests/NuGet.MSSigning.Extensions.FuncTest/MSSignCommandTestFixture.cs

```diff
@@ -106,8 +107,8 @@ private async Task<CertificateAuthority> CreateDefaultTrustedCertificateAuthorit
             _trustedTimestampRoot = TrustedTestCert.Create(
                 rootCertificate,
                 StoreName.Root,
-                StoreLocation.LocalMachine);
-
+                (RuntimeEnvironmentHelper.IsWindows || RuntimeEnvironmentHelper.IsMacOSX) ? StoreLocation.LocalMachine : StoreLocation.CurrentUser);
+            
```

Comment:
nit: extra whitespace. remove

Reply:
The background is https://github.com/dotnet/runtime/blob/master/docs/design/features/cross-platform-cryptography.md#x509store, added in the comments. Thanks!